spec/models/custom_attribute_spec.rb

```diff
@@ -10,8 +10,4 @@
   it "returns the value type of DateTime custom attributes" do
     expect(time_custom_attribute.value_type).to eq(:datetime)
   end
-
-  it "returns the value type of Fixnum custom attributes" do
-    expect(int_custom_attribute.value_type).to eq(:fixnum)
-  end
```

Comment:
cc @zgalor do you know if/how `CustomAttribute#value_type` is being used?
It's defined as `serialized_value.class.to_s.downcase.to_sym` :-(
I doubt removing the spec is enough.  We likely want either backward compat (return :fixnum on 2.4), or forward compat (switch to :integer on <=2.3), or remove the functionality.

API is only accepting (on POST?) a whitelist for field_type: https://github.com/ManageIQ/manageiq/search?utf8=%E2%9C%93&q=ALLOWED_API_VALUE_TYPES
https://github.com/ManageIQ/manageiq/blob/caa745c49/app/controllers/api/providers_controller.rb#L90
Not sure if/how `field_type` interacts with `value_type` and `serialized_value`.  Do we expose value_type on GET?